### ISSUE TITLE
stream resets when disconnecting Bluetooth device

### ISSUE THREAD

**Issue by chilliger** [mediathekview/zapp#330]
Disconnecting my Bluetooth buds causes the stream to reset. Stream just shows black screen and you lose the position.

I would expect that it just gets paused.

**Comment by cemrich** [comment]
I cannot reproduce this issue on Android 13. Which device, Android version and version of Zapp are you using?

**Comment by cemrich** [comment]
Yes, that would be helpful. Are you turning off your headphones or do they have connectivity issues?

**Comment by chilliger** [comment]
Turning them off.

**Comment by chilliger** [comment]
reproduced and filtered by keyword "zapp". Is this enough?
[logcat_11-05-2022_13-55-21.txt](https://github.com/mediathekview/zapp/files/9943903/logcat_11-05-2022_13-55-21.txt)

**Comment by cemrich** [comment]
The MediaSession receives a KEYCODE_MEDIA_STOP event, which may cause the stream to completely stop. This is tricky - I have to look into it. But I suspect the headphones to be the main problem here.
